INSTALL.md

```diff
@@ -156,3 +156,75 @@ Example:
 ```
 Lumen (5.3.2) (Laravel Components 5.3.*)
 ```
+
+## Installing ThesauRex as part of [Spacialist](https://github.com/eScienceCenter/Spacialist)
+
+As ThesaurRex is a crutial part of the [Spacialist](https://github.com/eScienceCenter/Spacialist) platform, it can be installed directly within the Spacialist environment as well. This part only contains changes that must be made when installing ThesauRex as part of Spacialist against a standalone installation.
+
+2. `cd` into your Spacialist Repository and clone the ThesauRex Repository into it
+
+   ```bash
+   git clone https://github.com/eScienceCenter/ThesauRex thesaurex
+   ```
+
+### Proxy Setup
+
+2. Add one entry for your spacialist and one for thesaurex to your hosts file
+
+   ```bash
+    sudo nano /etc/hosts
+    # Add an entry to "redirect" a domain to your local machine (localhost)
+    127.0.0.1 project.spacialist # or anything you want
+    127.0.0.1 project.thesaurex # or anything you want
+   ```
+
+3. Add an additional configuration to the vHost of you apache that you created while installing Spacialist
+
+   ```bash
+    cd /etc/apache2/sites-available
+    sudo nano thesaurex-lumen.conf
+   ```
+
+    Paste the following snippet into the file:
+   ```apache
+    <VirtualHost *:80>
+      ServerName project.spacialist
+      ServerAdmin webmaster@localhost
+      DocumentRoot /var/www/html/spacialist/lumen/public
+
+      DirectoryIndex index.php
+
+      <Directory "/var/www/html/spacialist/lumen/public">
+        AllowOverride All
+        Require all granted
+      </Directory>
+    </VirtualHost>
+
+   <VirtualHost *:80>
+      ServerName project.thesaurex
+      ServerAdmin webmaster@localhost
+      DocumentRoot /var/www/html/spacialist/thesaurex/lumen/public
+
+      DirectoryIndex index.php
+
+      <Directory "/var/www/html/spacialist/thesaurex/lumen/public">
+        AllowOverride All
+        Require all granted
+      </Directory>
+    </VirtualHost>
+   ```
+
+4. Add the proxy route to your default vHost file (e.g. `/etc/apache2/sites-available/000-default.conf`) with reference to the path to the api-folder of your ThesauRex Installation and the name you gave within your host file (see 2.)
+
+   ```apache
+    ProxyPass "/spacialist/thesaurex/api" "http://project.thesaurex"
+    ProxyPassReverse "/spacialist/thesaurex/api" "http://project.thesaurex"
+   ```
+
+### Configure Lumen
+
+As you have configured your `.env` file with your DB connection during the installation of Spacialist (see [Spacialist/INSTALL.md](https://github.com/eScienceCenter/Spacialist/blob/master/INSTALL.md)) its sufficient to just set a soft link to the `.env` file of Spacialist
+
+```bash
+ln -s /var/www/html/spacialist/lumen/.env /var/www/html/spacialist/thesaurex/lumen/.env
+```
```

Comment:
Here, only changes to the instructions above are listed, therefor no bower, composer or migrations